packages/create-waku/src/index.ts

```diff
@@ -162,9 +161,11 @@ Options:
 }
 
 async function notifyUpdate() {
-  // keep original require to avoid
-  //  bundling the whole package.json by `@vercel/ncc`
-  const packageJson = createRequire(import.meta.url)('../package.json');
+  const packageJson = JSON.parse(
+    readFileSync('../package.json', {
+      encoding: 'utf8',
+    }),
```

Comment:
We use the shorthand everywhere else.
```suggestion
    readFileSync('../package.json', 'utf8'),
```